_content/monographs/the-paccekabuddha-buddhist-ascetic_kloppenborg.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "The Paccekabuddha:
+A Buddhist Ascetic. A Study of the Concept of the 
+Paccekabuddha in Pali Canonical and Commentarial Literature"
+authors:
+  - "Ria Kloppenborg"
+external_url: "https://www.bps.lk/olib/wh/wh305_Kloppenborg_Paccekabuddha-Concept-In-Pali-Canon-Commentary.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1UjtEVCir4HE8CdOx3Zrv1HvVinVuIQZa/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1orAgVjHpl64YY0z7WPaIrNY1Wsnrcc2g/view?usp=drive_link"
+course: cosmology
+tags:
+  - meditation
+  - asceticism
+  - renunciation
+  - solitude
+  - sage
+year: 1983
+oclc: 17511360
+publisher: BPS
+address: "Kandy, Sri Lanka"
+number: 305, 306, 307
+pages: 100
+---
+
+> The concept of the Paccekabuddha presented the
+opportunity to include pre-Buddhist recluses and seers in
+Buddhism and in doing so it continued these pre-Buddhist
+traditions. In this respect it becomes clear why
+Paccekabuddhas are referred to in the scriptures with all
+other terms that could be used to denote ascetics: muni, isi,
+samaṇa, tāpasa, jaṭila, terms which emphasise different
+aspects of asceticism
+
+An in-depth study of the Paccekabuddha as described in Pali Canonical and Commentarial Literature. This includes pre-Buddhist ideas of sages, i.e. munis, and the importance given to renunciation and solitude in the suttas. Also, this work looks at a Paccekabuddha's way of life and meditation practices, leading to nibbāna. As the Buddha tells us, "no one but me equals a paccekabuddha" (Isigili Sutta). 
```

Comment:
muni could be in italics as it's a perhaps rarer foreign word? Idk... Bhikkhu Bodhi tried to make it a thing in his Snp translation but it hasn't seemed to catch on.